content/guides/references/migration-guide.md

```diff
@@ -2,6 +2,345 @@
 title: Migration Guide
 ---
 
+## Migrating to Cypress version 12.0
```

Comment:
I'd like to add in here somewhere a section about retryability, linking to the rewritten guide.

> Cypress 12 has improved and changed the way we retry commands by introducing a new type of Cypress command: Queries. Many commands you're already familiar with are now queries, such as `.get()`, `.find()`, and `.children()`. You can learn more about queries - and the way they protect you from Detached From DOM errors - in our guide on [Retry-ability](https://deploy-preview-4835--cypress-docs.netlify.app/guides/core-concepts/retry-ability#Commands-Queries-and-Assertions).

> In most cases, you won't need to change your tests to take advantage of this functionality, but it does have one implication to be aware of, in the form of a new Best Practice for writing Cypress tests:

> - [Action commands](https://deploy-preview-4835--cypress-docs.netlify.app/guides/core-concepts/interacting-with-elements#Actionability) (such as `.click()`, `.type()`, and `.trigger()`) should be at the end of chains, not the middle.

> See the guide linked above for more details.